### ISSUE TITLE
MSSQL tests cannot run on M1 macbooks

### ISSUE THREAD

**Issue by edgao** [airbytehq/airbyte#14964]
base-normalization's MSSQL tests fail on my laptop. @grishick also verified that source-mssql tests fail locally.

I looked very briefly into this a while back (but apparently didn't actually file a ticket) and found that:
* The official MSSQL image doesn't support arm64 (https://github.com/microsoft/mssql-docker/issues/668)
* There's an IOT-focused version of mssql (`azure-sql-edge`), which _does_ publish an arm64 version, but it lacks certain features that we depend on

So... possible next steps:
* figure out how to get the mssql image running on m1 (?!)
* do more research and try to find a working docker image (??)
* stand up an mssql instance in RDS / Azure / whatever and distribute creds to developers (not ideal, shared environments can cause problems)
* something else?

**Comment by evantahler** [comment]
@edgao is this still the case?

**Comment by edgao** [comment]
yeah, afaict all of it's still true. Recent comments on https://github.com/microsoft/mssql-docker/issues/668 claim that the docker image we're using still doesn't have an m1-compatible version
